cirq/google/engine/engine_processor.py

```diff
@@ -277,29 +277,70 @@ def list_reservations(
 
     def get_schedule(
             self,
-            from_time: datetime.datetime = datetime.datetime.now(tz=utc),
-            to_time: datetime.datetime = datetime.datetime.now(tz=utc) +
-            datetime.timedelta(weeks=2),
-            time_slot_type: qenums.QuantumTimeSlot.TimeSlotType = None
+            from_time: Union[None, datetime.datetime, datetime.
+                             timedelta] = datetime.timedelta(),
+            to_time: Union[None, datetime.datetime, datetime.
+                           timedelta] = datetime.timedelta(weeks=2),
+            time_slot_type: Optional[qenums.QuantumTimeSlot.TimeSlotType] = None
     ) -> List[EngineTimeSlot]:
         """Retrieves the schedule for a processor.
 
         The schedule may be filtered by time.
 
         Time slot type will be supported in the future.
+
+        Args:
+            from_time: Filters the returned schedule to only include entries
+                that end no earlier than the given value. Specified either as an
+                absolute time (datetime.datetime) or as a time relative to now
+                (datetime.timedelta). Defaults to now (a relative time of 0).
+                Set to None to omit this filter.
+            to_time: Filters the returned schedule to only include entries
+                that start no later than the given value. Specified either as an
+                absolute time (datetime.datetime) or as a time relative to now
+                (datetime.timedelta). Defaults to two weeks from now (a relative
+                time of two weeks). Set to None to omit this filter.
+            time_slot_type: Filters the returned schedule to only include
+                entries with a given type (e.g. maintenance, open swim).
+                Defaults to None. Set to None to omit this filter.
+
+        Returns:
+            Schedule time slots.
         """
+        now = datetime.datetime.now()
```

Comment:
Please ensure that this works as expected in a colab environment. I expect that the timestamp is going to be skewed by the utc offset.

Reply:
Why would you expect dates to behave differently in colab? That would be a pretty serious problem with colab.

Reply:
In a new colab notebook the timezone seems to be set to UTC so you can't tell the difference between the two methods.